client/lib/signup/step-actions/index.js

```diff
@@ -508,7 +517,16 @@ export function addAddOnsToCart(
 	}
 
 	const newCartItems = cartItem.filter( ( item ) => item );
-	processItemCart( providedDependencies, newCartItems, callback, reduxStore, slug, null, null );
+	processItemCart(
+		providedDependencies,
+		newCartItems,
+		callback,
+		reduxStore,
+		slug,
+		null,
+		null,
+		null
```

Comment:
Maybe isFreeThemePreselected, themeSligWithRepo and lastKnownFlow can be defaulted to null in processItemCart's prototype ?